src/stores/PreferenceStore/PreferenceStore.ts

```diff
@@ -630,11 +629,11 @@ export class PreferenceStore {
         return this.preferences.get(PreferenceKeys.PERFORMANCE_STOP_ANIMATION_PLAYBACK_MINUTES) ?? DEFAULTS.PERFORMANCE.stopAnimationPlaybackMinutes;
     }
 
-    @computed get pvPreivewCubeSizeLimit(): number {
+    @computed get pvPreviewCubeSizeLimit(): number {
         return this.preferences.get(PreferenceKeys.PERFORMANCE_PV_PREVIEW_CUBE_SIZE_LIMIT) ?? DEFAULTS.PERFORMANCE.pvPreviewCubeSizeLimit;
     }
 
-    @computed get pvPreivewCubeSizeLimitUnit(): string {
+    @computed get pvPreviewCubeSizeLimitUnit(): string {
```

Comment:
Since this is deprecated, this preference should never be accessed. I suggest removing this computed property entirely to ensure this. It should probably also be removed from the defaults.

If you hardcode the string name, you could actually also remove it from the preference keys, but perhaps it would be safer to keep it there for now and add a comment marking it as deprecated.